src/screens/Settings/index.js

```diff
@@ -0,0 +1,14 @@
+import {StyleSheet, Text, View} from 'react-native';
+import React from 'react';
+
```

Comment:
spacing should be proper
first import React from 'react'; then remain packages should import